### PR TITLE
Black code format + other code adjustments

### PR DESCRIPTION
**Pull request by RealistikDash**
Used Black to format the code and manually made all constants upper case

### CODE REVIEW

**Review by spookybear0 — APPROVED** [review]
I only went through half but it looks good so far.

File: cron/rankcalc.py [hunk 1]
```diff
@@ -9,7 +9,7 @@ async def cron_calc_ranks() -> None:
     async with myconn.conn.cursor() as mycursor:
         await mycursor.execute(
             "SELECT extID FROM users WHERE extID IN (SELECT accountID FROM accounts WHERE privileges & %s AND isBot = 0) ORDER BY stars DESC",
-            (Permissions.authenticate,),
+            (Permissions.AUTH,),
```
Comment (ghost) [inline cron/rankcalc.py:12]: why the ,

File: handlers/rewards.py [hunk 2]
```diff
@@ -37,10 +37,10 @@ async def quests_handler(request: aiohttp.web.Request):
     resp1 = encode_base64(
         cipher_xor(
             f"bruhh:{user.user_id}:{chk_decoded}:{device_id}:{user.account_id}:{tomorrow()-get_timestamp()}:{quest_str[0]}:{quest_str[1]}:{quest_str[2]}",
-            XorKeys.quests,
+            XorKeys.QUESTS,
         )
     )
-    hashed = solo_gen3(resp1)
+    hashed = SOLO_gen3(resp1)
     response = f"bruhh{resp1}|{hashed}"
```
Comment (ghost) [inline handlers/rewards.py:44]: ok

File: helpers/levelhelper.py [hunk 3]
```diff
@@ -101,18 +101,18 @@ async def multi_gen(self, levels: list) -> str:
 
             Hash += f"{str(level.ID)[0]}{str(level.ID)[len(str(level.ID))-1]}{level.stars}{int(level.verified_coins)}"
 
-        return hash_sha1(Hash + CryptKeys.solo)
+        return hash_sha1(Hash + CryptKeys.SOLO)
 
-    def solo_gen(self, level_str: str):
-        """Port of genSolo from Cvolton's GMDPrivateServer."""
+    def SOLO_gen(self, level_str: str):
+        """Port of genSOLO from Cvolton's GMDPrivateServer."""
```
Comment (ghost) [inline helpers/levelhelper.py:107]: nice

File: cron/rankcalc.py (hunk 1 above)
Comment (RealistikDash) [reply cron/rankcalc.py:12]: Python needs that to distinguish between a tuple and using brackets like maths

**Pull request merged**